lms/djangoapps/teams/tests/test_views.py

```diff
@@ -387,7 +403,7 @@ def test_access(self, user, status):
 
     def test_naming(self):
         new_teams = [
-            self.post_create_team(data=self.build_team_data(name=name))
+            self.post_create_team(data=self.build_team_data(name=name), user=self.create_and_enroll_student())
```

Comment:
Is this necessary because now the creator is enrolled in the team, and we only allow a user to be in 1 team? I wasn't sure that any of our code enforces the single team constraint.